h2o-py/h2o/backend/cluster.py

```diff
@@ -218,16 +218,19 @@ def check_version(self, strict=False):
             if build_number_h2o is None or build_number_h2o == "unknown":
                 message = ("Version mismatch. H2O is version {0}, but the h2o-python package is version {1}. "
                            "Upgrade H2O and h2o-Python to latest stable version - "
-                           "http://h2o-release.s3.amazonaws.com/h2o/latest_stable.html"
+                           "http://h2o-release.s3.amazonaws.com/h2o/latest_stable.html.  To circumvent this error "
```

Comment:
I wonder if `hide` would be clearer for non-native english speaking users.

Reply:
You are right.  I use avoid instead of circumvent.